sys/include/ztimer.h

```diff
@@ -577,12 +680,18 @@ ztimer_now_t _ztimer_now_extend(ztimer_clock_t *clock);
  *          unrelated components are altered that change the systems's power
  *          management behavior.
  *
+ * @warning Make sure to call @ref ztimer_acquire(@p clock) before fetching
+ *          the clock's current time.
```

Comment:
```suggestion
```

There are many ways validity of now can be guaranteed. I don't think pointing acquire out extra helps a lot; there is a warning in place already to read-and-understand all the above. Users should not be encouraged to use this lightly; when we have a higher-level abstraction in place, let's point there instead.

Reply:
Good point. Once that abstraction is in place, I'd rather discourage the usage of `ztimer_now()` for application developers.